test/use-swr.test.tsx

```diff
@@ -1267,6 +1267,62 @@ describe('useSWR - suspense', () => {
     // 'suspense-7' -> undefined -> 'suspense-8'
     expect(renderedResults).toEqual(['suspense-7', 'suspense-8'])
   })
+
+  // hold render when suspense
+  it('should pause when key is falsy', async () => {
+    function SectionContent({ swrKey }) {
+      const { data } = useSWR(
+        () => (swrKey % 2 ? null : 'suspense-' + swrKey),
+        key =>
+          new Promise(res =>
+            setTimeout(() => res(key.replace('suspense-', '')), 100)
+          ),
+        {
+          suspense: true
+        }
+      )
+
+      return <div>{data}</div>
+    }
+
+    const Section = () => {
+      const [key, setKey] = useState(9)
+      const handleClick = () => setKey(key + 1)
+
+      return (
+        <div onClick={handleClick}>
+          <Suspense fallback={<div>fallback</div>}>
+            <SectionContent swrKey={key} />
+          </Suspense>
+        </div>
+      )
+    }
+
+    const { container } = render(<Section />)
+
+    async function clickAndWait(duration) {
+      fireEvent.click(container.firstElementChild)
+      return await act(
+        () => new Promise(resolve => setTimeout(resolve, duration))
+      )
+    }
+
+    expect(
+      container.firstElementChild.lastElementChild.textContent
```

Comment:
since `<Suspense>` might render inline styles onto children, then the html will look like

```html
<div style="display: none !important">10</div>
<div>fallback</div>
```
so I always queried the content of last element child here. it would be wether the fallback div or the children with returned data.

@shuding I'm not sure if  it's sth I implemented incorrectly or there's any other elegant way to work around. other suspense test cases seem don't have this problem